### ISSUE TITLE
6.6.3: wrong return code for when trying to create a duplicate subvolume

### ISSUE THREAD

**Issue by vojtechtrefny** [kdave/btrfs-progs#730]
When trying to create a new subvolume which already exists, `btrfs subvolume create` has return code `0` with 6.6.3:

```
# btrfs subvolume create /mnt/test/a
Create subvolume '/mnt/test/a'
# btrfs subvolume create /mnt/test/a
ERROR: target path already exists: /mnt/test/a
# echo $?
0
# btrfs --version
btrfs-progs v6.6.3
```

Works as expected with 6.6.2:

```
# btrfs subvolume create /mnt/test/a
ERROR: target path already exists: /mnt/test/a
# echo $?
1
# btrfs --version
btrfs-progs v6.6.2
```

**Comment by mbiebl** [comment]
I can confirm that the referenced commit fixes the libblockdev test suite failure.

**Comment by kdave** [comment]
Thanks, will be released in 6.7.
